ctapipe/containers.py

```diff
@@ -759,12 +748,11 @@ class SimulatedCameraContainer(Container):
     )
 
 
-class SimulatedEventContainer(Container):
+class SimulationSubarrayContainer(Container):
     shower = Field(
```

Comment:
Right now, simulationsubarrayContainer has a single subcontainer, shower. 

We could just attach the `shower` container at the `SubarrayEventContainer` to get rid of one level of indirection.